brave/browser-laptop
# add noScriptInfo checkboxes to allow scripts by origin

test plan:
1. disable scripts globally in about:preferences
2. go to vox.com. images should not appear.
3. click on the noscript icon
4. unselect everything except vox.com and vox-cdn.com and click one of the 'allow' buttons
5. images should appear
6. click on the noscript icon. you should see a bunch of origins but not vox.com and vox-cdn.com

fix #6431. adjusts the noScriptInfo dialog so that it can be used to allow scripts by origin, instead of all scripts on the page, when NoScript is globally enabled. removes the option to allow scripts persistently (since this can be done via the Bravery panel anyway)

auditors: @bbondy


![screen shot 2017-02-15 at 10 45 06 pm](https://cloud.githubusercontent.com/assets/549654/23003490/070f295a-f3e9-11e6-864b-7c13ef561a58.png)
